clients/src/main/resources/common/message/ShareFetchRequest.json

```diff
@@ -36,7 +36,9 @@
     { "name": "MinBytes", "type": "int32", "versions": "0+",
       "about": "The minimum bytes to accumulate in the response." },
     { "name": "MaxBytes", "type": "int32", "versions": "0+", "default": "0x7fffffff",
-      "about": "The maximum bytes to fetch.  See KIP-74 for cases where this limit may not be honored." },
+      "about": "The maximum bytes to fetch. See KIP-74 for cases where this limit may not be honored." },
+    { "name": "MaxRecords", "type": "int32", "versions": "0+",
+      "about": "The maximum number of records to fetch. This is a soft limit and records can exceed upto the end of fetch log batch." },
```

Comment:
I suggest for the second sentence "This limit can be exceeded for alignment of batch boundaries".

Reply:
Done.